Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:argon
+
+# Install node_modules into a different directory to avoid npm/npm#9863.
+RUN mkdir -p /srv/addons-frontend-node
+ADD package.json /srv/addons-frontend-node
+WORKDIR /srv/addons-frontend-node
+RUN npm install -g npm@3
+RUN npm install
+
+ADD . /code
```

Comment:
I think we should start using /srv for everything since /code is not a standard dir.